src/vs/code/electron-main/app.ts

```diff
@@ -405,6 +405,8 @@ export class CodeApplication extends Disposable {
 		// Setup Auth Handler
 		this._register(appInstantiationService.createInstance(ProxyAuthHandler));
 
+		this.registerWebRequestFilter();
```

Comment:
We already have a security related method `configureSession` above that we really call as first thing. Should we maybe move your new method into that one to have a single method for configuring security related things right from the beginning?